src/viam/examples/modules/CMakeLists.txt

```diff
@@ -12,58 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-add_executable(example_module
```

Comment:
This cmake logic has been split between `src/viam/examples/modules/tflite/CMakeLists.txt` and `src/viam/examples/modules/simple/CMakeLists.txt`.